adobe/spectrum-web-components
# chore: update Spectrum CSS dependencies

## Description
- adopt latest Spectrum CSS dependencies
- ship and leverage `sp-close-button` as needed
- update `sp-clear-button` to normalize with above
- refactor `sp-color-loupe` for latest Spectrum CSS delivery

## Motivation and context
Keeping up to date with Spectrum CSS.

## How has this been tested?

    1. Go [here](https://62b89dbe8ce2940ea25a90053dc54d43--spectrum-web-components.netlify.app/review/#ColorWheelStories/sized.png)
    2. Checkout the diffs
    1. Go [here](https://spectrum-css--spectrum-web-components.netlify.app/components/button/)
    2. See the docs for `ClearButton` and `CloseButton`
    3. See that there are not individual pages for the elements.

## Types of changes

## Checklist

## Best practices

This repository uses conventional commit syntax for each commit message; note that the GitHub UI does not use this by default so be cautious when accepting suggested changes. Avoid the "Update branch" button on the pull request and opt instead for rebasing your branch against `main`.
